addons/website/static/src/js/menu/edit.js

```diff
@@ -72,9 +72,18 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
     start: function () {
         var def = this._super.apply(this, arguments);
 
+        let navbar = $('nav#oe_main_menu_navbar').get(0);
```

Comment:
- const
- move it inside the if, don't define it outside if you don't need it
- suffix by `El`

Reply:
And don't use jquery to get the element if you are going to straigh away convert it to DOM